Makefile.org

```diff
@@ -418,12 +418,22 @@ clean:	libclean
 	rm -f $(LIBS)
 	rm -f openssl.pc libssl.pc libcrypto.pc
 	rm -f speed.* .pure
+	$(RM) test/newkey.pem test/testkey.pem test/testreq.pem
+	-$(RM) `find . -name '*.o' -a \! -path "./.git/*"`
```

Comment:
If test/Makefile cleans newkey.pem testkey.pem and testreq.pem in the
clean-target, then how about this:
```
dist:   
        $(PERL) Configure dist
        @$(MAKE) SDIRS='$(SDIRS)' clean
        @$(MAKE) TAR='$(TAR)' TARFLAGS='$(TARFLAGS)' $(DISTTARVARS) tar
```
Isn't that going to change the contents of the TAR file?